projects/packages/jetpack-mu-wpcom/src/features/wpcom-admin-bar/wpcom-admin-bar.php

```diff
@@ -159,6 +159,61 @@ function wpcom_replace_wp_logo_with_wpcom_all_sites_menu( $wp_admin_bar ) {
 }
 add_action( 'admin_bar_menu', 'wpcom_replace_wp_logo_with_wpcom_all_sites_menu', 11 );
 
+/**
+ * Adds the Cart menu to the WordPress admin bar.
+ *
+ * @param WP_Admin_Bar $wp_admin_bar The WP_Admin_Bar core object.
+ */
+function wpcom_add_shopping_cart( $wp_admin_bar ) {
+	// Include the shopping cart functionality from the specified path.
+	require_once WP_CONTENT_DIR . '/admin-plugins/wpcom-billing/shopping-cart.php';
```

Comment:
hmm... I don't think we have `admin-plugins` on Atomic. Is this limited only to Simple? 🤔

Reply:
Yep, getting a fatal: PHP Fatal error:  Uncaught Error: Failed opening required '/srv/htdocs/wp-content/admin-plugins/wpcom-billing/shopping-cart.php' (include_path='/:.') in jetpack-mu-wpcom/src/features/wpcom-admin-bar/wpcom-admin-bar.php:169

Reply:
I've had a look and it's not so trivial to make it work on Atomic sites, let's limit the scope of this PR to Simple sites so that we can ship this working code.

Reply:
Then we need to add a check here that returns early if we're on an Atomic site :)

Reply:
Updated